client/src/app/containers/Alert/useAlert.ts

```diff
@@ -0,0 +1,32 @@
+import { nanoid } from '@reduxjs/toolkit';
+import { useDispatch } from 'react-redux';
+
+import { actions } from './slice';
+import { AlertType } from './types';
+
+export interface AlertSetter {
+    (message: string, type: AlertType): string;
+}
```

Comment:
:mag: why does this need to be exported?

Reply:
I actually don't think we need this anymore. I will review its usage.